kafka-connect-s3/pom.xml

```diff
@@ -392,5 +425,12 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jenkins</id>
+            <properties>
+                <skipIntegrationTests>false</skipIntegrationTests>
```

Comment:
can not use the property defined above. It will hit NPE

Reply:
could you clarify what hits the NPE? Which property can't we use?

Reply:
if replace false with ${skipIntegrationTests}, hit NPE on jenkins